test_regress/t/t_assert_clock_event_bad.out

```diff
@@ -0,0 +1,6 @@
+%Error-UNSUPPORTED: t/t_assert_clock_event_bad.v:26:7: Unsupported: Clock event before property call and in its instantion
+                                                     : ... In instance t
+   26 |       @(negedge clk)
+      |       ^
+                    ... For error description see https://verilator.org/warn/UNSUPPORTED?v=latest
+%Error: Exiting due to
```

Comment:
I think this test should be called `*_unsup*` instead of `*_bad*`.

Reply:
I changed the name.